UI/Authentication.OidcDemo/Authentication.OidcDemo/Authentication.OidcDemo.Droid/ChromeCustomTabsProvider.cs

```diff
@@ -1,27 +1,19 @@
-﻿using System;
-using System.Threading;
-using System.Threading.Tasks;
-using Windows.Security.Authentication.Web;
-using AndroidX.Browser.CustomTabs;
+﻿using AndroidX.Browser.CustomTabs;
 using Uno.AuthenticationBroker;
 using Uno.UI;
 
 namespace Authentication.OidcDemo.Droid
 {
 	public class ChromeCustomTabsProvider : WebAuthenticationBrokerProvider
 	{
-		protected override async Task LaunchBrowserCore(
-			WebAuthenticationOptions options,
-			Uri requestUri,
-			Uri callbackUri,
-			CancellationToken ct)
+		public ChromeCustomTabsProvider()
 		{
 			var builder = new CustomTabsIntent.Builder();
 			var intent = builder.Build();
 			
 			intent.LaunchUrl(
 				ContextHelper.Current,
-				Android.Net.Uri.Parse(requestUri.OriginalString));
+				Android.Net.Uri.Parse(base.GetCurrentApplicationCallbackUri().ToString()));
```

Comment:
@aesteves900 Did you find this new technique from some documentation? I wonder if we should also update our docs based on that.

Reply:
I'm not found on the docuemntation, I just tried to preserve the same behaviour as before.

Reply:
@carldebilly does this look familiar?

Reply:
```suggestion
				Android.Net.Uri.Parse(base.GetCurrentApplicationCallbackUri().OriginalString));
```

Yep the sample has been done before this feature was implemented. That's the correct change to do. As soon you tested it and it still works...